.vscode/settings.json

```diff
@@ -11,6 +11,11 @@
     "editor.rulers": [100]
   },
   "C_Cpp.default.cStandard": "c23",
+  "C_Cpp.default.defines": [
+    "false=0",  // Defining true and false here is required since the C/C++ extension's
+    "true=1",   // intellisense does not support this feature of c23 yet.
```

Comment:
This seems risky. I thought true was !false

Reply:
I guess this is not for compilation so it probably doesn't matter.

Reply:
Exactly, I just got annoyed about the red squiggles. I am updating the comments around these to better indicate their temporary nature and clarify that they aren't used for compilation.

As for true=!false, I assume that would work here but from the C++ spec:
> A prvalue of type bool can be converted to a prvalue of type int, with false becoming zero and true becoming one.

So I figured defining true=1 was the easiest.